packages/ts/lit-form/src/index.ts

```diff
@@ -7,6 +9,20 @@ export * from './Validation.js';
 export * from './Validators.js';
 export * from './Validity.js';
 
+const m = {
+  /**
+   * Returns an iterator over item models in the array model.
+   *
+   * @param model - The array model to iterate over.
+   * @returns An iterator over item models.
+   */
+  items<M extends ArrayModel>(model: M): Generator<ArrayItemModel<M>, void, void> {
+    return model[_items]() as Generator<ArrayItemModel<M>, void, void>;
+  },
+};
+
+export default m;
```

Comment:
Let's not use default export from the package index here — it's the Form library, not the models library. Does not feel semantically correct to export the model utils as the default asset here.

Reply:
Ah, right, didn't think about that. Thanks